### ISSUE TITLE
No check enabled for overriding ABC.abstractmethod

### ISSUE THREAD

**Issue by laszukdawid** [microsoft/pyright#644]
**Describe the bug**
Overriding an abstract class (abc.ABC) doesn't seem to trigger checks on the child class. I don't see checks on neither existence of the abstract method (abc.abstractmethod), nor its arguments. There was a similar issue (#470 ) reported though it was referring to multi-inheritace and presumably it's fixed. In my case that's a direct inheritance of a single abstract class.

**To Reproduce**
file: `src/__init__.py`

```python
import abc

class A(abc.ABC):
    @abc.abstractmethod
    def get(self, arg1: str, arg2: bool):
        pass
```
file: `src/run.py`

```python
from . import A

class B(A):
    def not_get(self): print('Ok')

class C(A):
    def get(self, arg1: str): print('Still Ok')
```

**Expected behavior**
In both B and C cases there should be type check.

**VS Code extension or command-line**
I'm running `pyright` v1.1.34 on VS Code v1.44.2 on Ubuntu 16.04 in a custom environment.

**Comment by erictraut** [comment]
Do you have the "reportIncompatibleMethodOverride" rule enabled? By default, it's disabled. When I enable this setting, it correctly reports an error here.

You can enable this in the config file (pyrightconfig.json) or on a per-file basis using a comment: `# pyright: reportIncompatibleMethodOverride=true`. Or you can enable all rules by enabling "strict" mode.

As for reporting that an abstract method is not overridden in a subclass, that's reported at the time you attempt to instantiate the subclass. Using your example above, if you try to instantiate `B` (e.g. `b = B()`), Pyright will report an error 'Cannot instantiate abstract class "B"'.

**Comment by laszukdawid** [comment]
Thanks for the prompt response. Updating `pyrightconfig.json` didn't do much but changing to `strict` mode or adding mentioned comment did the trick.
Clearly need to go through the doc slowly.
